grant.md

```diff
@@ -1,19 +1,245 @@
 ---
 title: GRANT
-toc: true
+toc: false
 ---
 
-## Description
+The `GRANT` [statement](sql-statements.html) grants privileges on one or more databases or tables to one or more users. 
+
+The user granting privileges must have the `GRANT` privilege on the target databases or tables.  
+
+<style>
+div#toc ul {
+    max-width: 50%;
+}
+</style>
+
+<div id="toc"></div>
 
 ## Synopsis
 
 {% include sql/diagrams/grant.html %}
 
-## Privileges
+## Supported Privileges
+
+CockroachDB supports the following privileges at the database and table levels. For details about privileges required by SQL statements not explicitly covered here, see the [SQL statement](sql-statements.html) documentation.
+
+- [`ALL`](#all)
+- [`CREATE`](#create)
+- [`DROP`](#drop)
+- [`GRANT`](#grant)
+- [`SELECT`](#select)
+- [`INSERT`](#insert)
+- [`DELETE`](#delete)
+- [`UPDATE`](#update)
+
+### `ALL`
+
+Level | Description
+------|------------
+Database | The user has all privileges on all tables in the database.
```

Comment:
This is a bit confusing. The `Database` level grants privileges on that database only. Future tables will inherit the privileges, but that's it.
Some privileges at the database level also mean something specific (eg: `CREATE` at the database level is needed to create a table inside that database) while some mean nothing at all at the database level (eg: `SELECT` is only useful at the table level).

I think it might be good to make the privilege inheritance clearer. You do mention it in the future tables section, but I would go ahead and spell out that a new table inherits the database's privileges.